addons/account/models/chart_template.py

```diff
@@ -941,7 +941,7 @@ def _get_tag_mapper(self, template_code):
             ('applicability', '=', 'taxes'),
             ('country_id', '=', self._get_chart_template_mapping()[template_code]['country_id']),
         ])}
-        return lambda *args: [tags[re.sub(r'\s+', ' ', x.strip())] for x in args]
+        return lambda *args: [tags[re.sub(r'\s+', ' ', x.strip())] if not re.match(r"^(\w+\.\w+,)*\w+\.\w+$", x) else x for x in args]
```

Comment:
```suggestion
        return lambda *args: [tags[re.sub(r'\s+', ' ', x.strip())] if not re.match(r"^\w+\.\w+$", x) else x for x in args]
```
The terms are split here (see line 954), so I don't think keeping the first part of the regex makes sense, does it ? (I didn't test)